added faq and link from faq page (#4334)
DataDog/documentation: content/en/monitors/faq/why-did-my-monitor-settings-change-not-take-effect.md

---
title: Why Did My Monitor Settings Change Not Take Effect?
kind: faq

---

> We will keep monitor groups available in the UI for 24 hours.  If you do not have "No Data" alert settings enabled and your group for a metric monitor stops reporting data, it will persist on the monitor status page until it ages out, though that group will stop being evaluated after a short absence (specific timing depends on your settings).

Reviewer: ```suggestion
Datadog keeps monitor groups available in the UI for 24 hours.  If you do not have *No Data* alert settings enabled and your group for a metric monitor stops reporting data, it will persist on the monitor status page until it ages out, though that group will stop being evaluated after a short absence (specific timing depends on your settings).
```

> For event monitors, however, we will also keep groups for evaluations for at least 24 hours. This means that if a monitor is updated and the groups are changed in the query, some old groups may persist. If you must change the group settings on your event monitor, you may want clone or create a new monitor to reflect your new groups.  Alternatively, you can mute them if you'd like to maintain the monitor but silence any alerts that would result from the changes.

Reviewer: ```suggestion
For event monitors, however, Datadog also keeps groups for evaluations for at least 24 hours. This means that if a monitor is updated and the groups are changed in the query, some old groups may persist. If you must change the group settings on your event monitor, you may want clone or create a new monitor to reflect your new groups.  Alternatively, you can mute them if you would like to maintain the monitor but silence any alerts that would result from the changes.
```
